python.html.markdown

```diff
@@ -145,8 +145,10 @@ bool("")  # => False
 print "I'm Python. Nice to meet you!"
 # Python also has a print function, available in versions 2.7 and 3...
 # but for 2.7 you need to add the import (uncommented):
-# from __future__ import print_function
+from __future__ import print_function
```

Comment:
I would say let's just get rid of the `__future__` discussion entirely. There are lots of Python3 features that can be enabled from a `__future__` import, kind of odd that we're singling out printing.